storage/diskmetricstore.go

```diff
@@ -437,6 +460,31 @@ func copyMetricFamily(mf *dto.MetricFamily) *dto.MetricFamily {
 	}
 }
 
+// groupingKeyFor creates a grouping key from the provided map of grouping
```

Comment:
Usually I think it would be nice to include a comment for what the format of the type is since we don't have a `type groupingKey struct` but in this case the format is pretty clear from the function implementation and the tests. Up to you if you want to clarify the format or uniqueness of the key for a given set of label names/values here.

Reply:
I added an explanation how the grouping key is created and that it is unique.